src/test/java/com/gudimov/movieland/dao/jdbc/JdbcMovieDaoITest.java

```diff
@@ -29,4 +29,10 @@ public void getRandom() throws Exception {
         assertNotEquals(randomMovie.size(), 0);
     }
 
+    @Test
+    public void getByGenreId() throws Exception {
+        List movieByGenreId = jdbcMovieDao.getByGenreId(1);
+        assertNotEquals(movieByGenreId.size(), 0);
```

Comment:
consider getting real id from db first for future ITests, overwise it will fail if genre with id = 1 disapear from db )